Makefile

```diff
@@ -26,6 +27,7 @@ exe: _output/bin/limactl$(exe)
 .PHONY: binaries
 binaries: clean \
 	_output/bin/lima \
```

Comment:
I guess this line can be removed now

Reply:
I want to have both of them, the shell variants are preferred - the other is just a fallback

I _don't_ want to spend time porting anything like nerdctl.lima to it, but made an exception for "lima"